mindmeld/components/dialogue.py

```diff
@@ -820,6 +821,19 @@ def _validate(self, request, slot):
 
             else:
                 # gazetteer validation
+
+                # payload format for entity feature extractors:
+                # tuple(query (Query Object), list of entities, entity index)
+                if request.entities:
+                    query = self._extract_query_features(
+                        request.entities[0]["value"][0]["cname"]
+                    )
+
+                if not query:
+                    query = self._extract_query_features(text)
+
+                formatted_payload = (query, [query], 0)
```

Comment:
Tiny nit: does this look neater this way?

```suggestion
                extracted_feature = entity_features.extract_in_gaz_features()(
                    (query, [query], 0), gazetteer
                )
```

Reply:
Updated the order slightly, but kept the extra variable. Just felt easier to read and explain to anyone who reads the code later.